metashare/api/migrations/0046_auto_20191223_1926.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.9 on 2019-12-23 19:26
+
+import re
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0045_repository_branch_name"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="repository",
+            name="branch_name",
+            field=models.CharField(
+                blank=True,
+                default="master",
```

Comment:
At this stage we could probably just modify migration `0045` instead of adding this new one?